fitness_functions/publish.py

```diff
@@ -1,2 +1,61 @@
+import os
+import ast
+import sqlite3
+from datetime import datetime
+from itertools import islice
+import matplotlib.pyplot as plt
+from sklearn import preprocessing
+from fitness_functions.run import run
+
+
+# Function to normalise the graph data
+def normalise(fitness_dictionary):
+    for key, value in islice(fitness_dictionary.items(), 1, None):
+        fitness_dictionary[key] = [round(n * 100, 0) for n in preprocessing.normalize([value])[0]]
+    return fitness_dictionary
+
+
 def publish(project_path, code_path):
```

Comment:
We can remove the code_path variable I think?

```suggestion
def publish(project_path):
```